### PR TITLE
Disabled packing for console and xdr projects

### PR DESCRIPTION
**Pull request by cuongph87**
Updated setting so a build runs on a push to a PR.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: stellar-dotnet-sdk-xdr/stellar-dotnet-sdk-xdr.csproj [hunk 1]
```diff
@@ -19,6 +18,7 @@
     <PackageTags>'.NET Core' 'Stellar' 'Horizon' 'Blockchain' 'Crypto' 'Distributed Ledger'</PackageTags>
     <ReleaseNotes>https://github.com/elucidsoft/dotnet-stellar-sdk/releases</ReleaseNotes>
```
Comment (jopmiddelkamp) [inline stellar-dotnet-sdk-xdr/stellar-dotnet-sdk-xdr.csproj:19]: Can't we remove all the NuGet package info from the csproj?

File: .github/workflows/publish_nuget.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
     - 'master'
+  pull_request_target:
+    types: [synchronize]
```
Comment (jopmiddelkamp) [inline .github/workflows/publish_nuget.yml:8]: Should it not also trigger when we open a PR?

```suggestion
    types:
      - opened
      - synchronize
```

**Review by jopmiddelkamp — CHANGES_REQUESTED** [review]

File: .github/workflows/publish_nuget.yml (hunk 2 above)
Comment (cuongph87) [reply .github/workflows/publish_nuget.yml]: I didn't know "opened" wasn't a type of "synchronize".

File: .github/workflows/publish_nuget.yml [hunk 3]
```diff
@@ -4,6 +4,8 @@ on:
   push:
     branches:
     - 'master'
+  pull_request_target:
+    types: [opened, synchronize]
```
Comment (jopmiddelkamp) [inline .github/workflows/publish_nuget.yml:8]: Why do we use this different notation for this. Maybe also change the rest then.

```yml
  workflow_dispatch:
  push:
    branches: [master]
  pull_request_target:
    types: [opened, synchronize]
  release:
    types: [published]
```

But I think I prefer the `-` notation for readability
```yml
  workflow_dispatch:
  push:
    branches:
    - master
  pull_request_target:
    types:
    - opened
    - synchronize
  release:
    types:
    - published
```

But just pick the one you prefer as long as they all are the same.

File: stellar-dotnet-sdk-console/stellar-dotnet-sdk-console.csproj [hunk 4]
```diff
@@ -4,6 +4,10 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
     <Nullable>enable</Nullable>
+    <IsPackable>false</IsPackable>
+    <PackageId />
```
Comment (jopmiddelkamp) [inline stellar-dotnet-sdk-console/stellar-dotnet-sdk-console.csproj:8]: Can we not completely remove these empty elements? If not please resolve.

**Review by jopmiddelkamp — CHANGES_REQUESTED** [review]

File: .github/workflows/publish_nuget.yml [hunk 5]
```diff
@@ -4,9 +4,13 @@ on:
   push:
     branches:
     - 'master'
```
Comment (jopmiddelkamp) [inline .github/workflows/publish_nuget.yml:6]:
```suggestion
    - master
```

**Review by jopmiddelkamp — APPROVED** [review]

**Review by jopmiddelkamp — APPROVED** [review]

**Pull request merged**